content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -188,6 +188,43 @@ The following plugins support `WaitForFirstConsumer` with pre-created Persistent
 and pre-created PVs, but you'll need to look at the documentation for a specific CSI driver
 to see its supported topology keys and examples.
 
+:::note
+   If you choose to use `waitForFirstConsumer` in storage classes, do not use `nodeName` in the Pod spec
+   to specify node affinity. If `nodeName` is used in this case, then PVC will remain in `pending` state. For more details refer https://github.com/openebs/openebs/issues/2915.
```

Comment:
I don't think we should refer to issues outside of the kubernetes repo. Maybe just say something like `the scheduler will be bypassed and the PVC will remain...`